nerdtree_plugin/menu_sys_open.vim

```diff
@@ -14,7 +14,14 @@ function! s:callback_name()
 endfunction
 
 function! s:callback()
-    if exists('*vimproc#open')
+    if !exists("g:netrw_nogx") 
+	let oldpwd = getcwd()
+        let ret = g:NERDTreeFileNode.GetSelected().path.changeToDir()
+	if !ret
+	    exe "normal gx"
+	    exe "cd" . oldpwd 
+	endif
+    elseif exists('*vimproc#open')
```

Comment:
My only concern here is that NERDTree would be tightly coupled to another plugin, over which we have no control. I think this coupling is best handled in a user's `.vimrc` file.